components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -81,15 +74,73 @@ export default function () {
                         </a>
                     </span>
                 }
+                disabled={!!project.settings?.allowUsingPreviousPrebuilds}
```

Comment:
**thought**: Disabling controls from a different section is a sign we could group better things in this page. Thoughts on grouping the top controls?

| BEFORE | AFTER (Disabled) | AFTER (Enabled) |
|-|-|-|
| <img width="1440" alt="Screenshot 2022-11-07 at 7 12 26 PM" src="https://user-images.githubusercontent.com/120486/200372647-1382cc8d-bf2a-4e36-8f1e-e3a6e54a5de3.png"> | <img width="1440" alt="Screenshot 2022-11-07 at 6 58 04 PM" src="https://user-images.githubusercontent.com/120486/200372613-2ea758bf-64f7-423f-b54c-069490a8e48d.png"> | <img width="1440" alt="Screenshot 2022-11-07 at 7 12 13 PM" src="https://user-images.githubusercontent.com/120486/200372617-168527ef-2174-482d-bc1e-5227e1370c55.png"> |